engine/impl/Playlist.php

```diff
@@ -162,39 +164,86 @@ public function updatePlaylist($playlist, $date, $time, $description, $airname)
     }
 
     public function getTrack($id) {
-        $query = "SELECT tag, artist, track, album, label, id FROM tracks " .
+        $query = "SELECT created, tag, artist, track, album, label, id FROM tracks " .
                  "WHERE id = ?";
         $stmt = $this->prepare($query);
         $stmt->bindValue(1, (int)$id, \PDO::PARAM_INT);
         return $this->executeAndFetch($stmt);
     }
     
     public function getTracks($playlist, $desc = 0) {
-        $query = "SELECT tag, artist, track, album, label, id FROM tracks " .
+        $query = "SELECT created, tag, artist, track, album, label, id FROM tracks " .
```

Comment:
New column 'created' needs to go to the end of the existing SELECT columns.

Reply:
done. please add this to the contributing guidelines.